kettle-columnstore-bulk-exporter-plugin/src/main/java/com/mariadb/columnstore/api/kettle/KettleColumnStoreBulkExporterStepDialog.java

```diff
@@ -189,6 +197,22 @@ public void modifyText( ModifyEvent e ) {
       }
     };
 
+    // Fill the set of reserved words from file reserved_words.txt
+    if(getClass().getResource(RESERVED_WORDS_FILENAME) == null){
+      logError("can't access the reserved words file " + RESERVED_WORDS_FILENAME);
+    } else {
+      try {
+        InputStream is = getClass().getResourceAsStream(RESERVED_WORDS_FILENAME);
```

Comment:
sorry missed this last time, but you should close the inputstream (and reader) as you are probably leaving a file handle open.